projects/models.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  pkgs,
+  sources,
+}:
+let
+  yants = import sources.yants { };
+
+  inherit (yants)
+    string
+    list
+    option
+    attrs
+    ;
+in
+rec {
+  project =
+    p: with p; {
+      name = string name;
+      metadata = with metadata; {
+        summary = string summary;
+        fund = string fund;
+        status = string status;
```

Comment:
Funds can be calculated from subgrants, and we should probably have those as a list of enum here.